src/main/resources/com/cloudbees/jenkins/plugins/bitbucket/endpoints/BitbucketServerEndpoint/manage-hooks-detail.jelly

```diff
@@ -1,8 +1,6 @@
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form">
   <f:entry title="${%Webhook implementation to use}" field="webhookImplementation">
```

Comment:
You should be able to into the config-detail.jelly instead

Reply:
Oh I did not see it was put into the context of the following on line 9 😅 https://github.com/jenkinsci/bitbucket-branch-source-plugin/blob/7020e28593d61ac486b147bdb5253658998d7d86/src/main/resources/com/cloudbees/jenkins/plugins/bitbucket/endpoints/AbstractBitbucketEndpoint/config.jelly#L5-L10